### DISCUSSION TITLE
The only user in a Odoo v16.3 DB set as non admin

### DISCUSSION THREAD

**Discussion by govind65** [OCA/odoo-community.org#99] [Q&A]
Hi all, I am new to Odoo and trying to find my way through with a one free app subscription. Trying to find a way to change the date format from MM/DD/YYYY to DD/MM/YYYY i mistakenly managed to save the database with my one and only user set to Administrator/Access rights. Now I find myself in a situation where I am not able to make any changes or delete the DB, while I am not able to update teh user rights to Administrator/Settings to regain admin rights.

Is there a way I can delete this DB and start fresh? or regain the admin rights?

**Comment by theangryangel** [comment]
If you can get into the shell then you could fix it, however, given you’re running 16.3 I assume this is an Odoo (the company) hosted instance? If it’s Odoo Online hosting, log a support ticket with Odoo (the company) - the OCA isn’t able to really help here (the OCA is a separate entity).

**Reply by govind65** [reply]
Many thanks for the prompt reply. You are correct in saying that it is Odoo online hosting version. How to open a support ticket?

**Reply by govind65** [reply]
support ticket opened, awaiting response. I hope this is not a chargable ticket as it is actually a limitation/bug of the system.

**Comment by Shide** [comment]
Date format changes depending on which language you are choosing for the current user. 

The selected language also has this kind of information, so it's better to focus on which language is going to use every user instead of thinking about the date format.
If an English user downloads a report, the date will be shown as YYYYMMDD, but if an Spanish user downloads a report, the date will be displayed as DDMMYYYY.

Also you can modify the date format in the Language records, but I strong recommend you to not modify this.

**Reply by govind65** [reply]
From the available English options that show to me, I have selected English AU. which normally should have DD/MM/YYYY. But, still displays in MM/DD/YYYY. 

I am using the cloud version so changing any more than the language is not possible I am guessing.

**Reply by Shide** [reply]
If you think it's an error, you should create a ticket on Odoo to fix it. If you can provide some legal/government information about that, it will be fast.

**Reply by gdgellatly** [reply]
No, that isn't what happens. When you change language in a fresh install there are 6 records or so that retain the original language and need manual updating. These are the 3 users (2 hidden) and then I can't remember, partners or employees, also website. Once you have ensured they all have the language you want, you can go to languages and deactivate.

It is much easier to get correct in first place.
